packages/terra-paginator/src/_PaginatorButton.jsx

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import { disableFocusActiveStyles, disableFocusOnBlur, enableFocusActiveStyles } from './_paginationUtils';
+import styles from './Paginator.module.scss';
+
+const cx = classNames.bind(styles);
+
+const propTypes = {
+  /**
+   * Sets aria-current attribute.
+   */
+  ariaCurrent: PropTypes.oneOfType([
+    PropTypes.bool,
+    PropTypes.string,
+  ]),
+  /**
+   * Sets aria-disabled attribute.
+   */
+  ariaDisabled: PropTypes.bool,
+  /**
+   * Sets aria-label attribute.
+   */
+  ariaLabel: PropTypes.string,
+  /**
+   * Child elements passed in to buttons.
+   */
+  children: PropTypes.node,
+  /**
+   * Passed in CSS className.
+   */
+  className: PropTypes.string,
+  /**
+   * Callback function triggered when clicked.
+   */
+  onClick: PropTypes.func,
+};
+
+function PaginatorButton(props) {
+  let shouldShowFocus = true;
+
+  const handleFocus = (event) => {
+    if (shouldShowFocus) event.currentTarget.setAttribute('data-focus-styles-enabled', 'true');
+  };
+
+  const handleMouseDown = () => {
+    // Prevent button from showing focus styles when clicked
+    shouldShowFocus = false;
+    // Wait until after onFocus has been triggered on browsers where it will get triggered for click
+    setTimeout(() => { shouldShowFocus = true; });
```

Comment:
Should this timeout value have a time value as well, not just the function to execute when the timeout is reached

When comparing to terra-button, should we be preventing default on the event? https://github.com/cerner/terra-core/blob/master/packages/terra-button/src/Button.jsx#L212

Reply:
The timeout doesn't appear to be necessary because of the way the events are handled now, and styling works as expected without. Prevent default added to event - [3c4f04a](https://github.com/cerner/terra-core/pull/2686/commits/3c4f04a82bd80d77dc4fc4f35ba36c5d85deb003)